Sources/MapboxMaps/Viewport/Transitions/Default/DefaultViewportTransition.swift

```diff
@@ -26,13 +26,21 @@ extension DefaultViewportTransition: ViewportTransition {
     public func run(to toState: ViewportState,
                     completion: @escaping (Bool) -> Void) -> Cancelable {
         let resultCancelable = CompositeCancelable()
+        var animation: DefaultViewportTransitionAnimationProtocol?
```

Comment:
If we know that we are going to intialize animation later before we use it, can we use `!` here instead?